### PR TITLE
Improve Example app to no zoom animation after active navigation

### PR DESCRIPTION
**Pull request by S2Ler**
# Before

https://user-images.githubusercontent.com/413986/143578299-2e063cc4-7204-45db-8f8f-a6df47cdcaaa.mp4

# After

https://user-images.githubusercontent.com/413986/143578129-acce1415-7312-497c-8105-acd23d448c6b.mp4

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -186,6 +187,14 @@ class ViewController: UIViewController {
         
         setupGestureRecognizers()
         setupPerformActionBarButtonItem()
+        if let lastKnownActiveNavigationLocation = lastKnownActiveNavigationLocation {
+            /// Empirical zoom level that produce no zoom animation after active navigation dismiss
+            let zoom = 14.375349173038527
```
Comment (MaximAlien) [inline Example/ViewController.swift:192]: Why not remember last known `CameraOptions`? (it'll include both last known coordinate and zoom level)

**Comment by MaximAlien** [comment]
Also getting build error:
```
/Users/distiller/project/Example/ViewController.swift:194:66: error: cannot convert value of type 'Double' to expected argument type 'CGFloat?'
                                                           zoom: zoom),
```

File: Example/ViewController.swift (hunk 1 above)
Comment (S2Ler) [reply Example/ViewController.swift:192]: Thanks for suggestion, I'll try

Comment (S2Ler) [reply Example/ViewController.swift]: Done

**Comment by S2Ler** [comment]
> Also getting build error:
> 
> ```
> /Users/distiller/project/Example/ViewController.swift:194:66: error: cannot convert value of type 'Double' to expected argument type 'CGFloat?'
>                                                            zoom: zoom),
> ```

Should be ok now

File: Example/ViewController.swift [hunk 2]
```diff
@@ -647,4 +654,11 @@ extension ViewController: NavigationViewControllerDelegate {
         dismissActiveNavigationViewController()
         clearNavigationMapView()
     }
+
+    func navigationViewController(_ navigationViewController: NavigationViewController,
```
Comment (MaximAlien) [inline Example/ViewController.swift:658]: Shall we remove `lastKnownActiveNavigationLocation` usage?

**Review by MaximAlien — APPROVED** [review]

**Pull request merged**